packages/typescript-eslint/package.json

```diff
@@ -51,6 +51,7 @@
     "check-types": "npx nx typecheck"
   },
   "dependencies": {
+    "@eslint/config-helpers": "^0.2",
```

Comment:
ugh an `0.x` version sucks cos it means that every minor release we'll be forced to update our version or else users will be blocked.

ESLint itself depends on this package so in some future when we drop v8 and older v9 versions -- we would be able to make this a peer dependency and just rely on the version installed that way.